isofit/radiative_transfer/luts.py

```diff
@@ -58,7 +59,10 @@ def initialize(
         List of keys to default to zeros as the fill value instead of NaNs
     """
     # Initialize with all lut point names as dimensions
-    ds = xr.Dataset(coords={"wl": wl} | lut_grid)
```

Comment:
Python 3.11 is the first version to support merging dictionaries with the `|` operator. Use `dict.update()` for broader compatibility instead.